uktrade/data-hub-frontend
# Reactify OMIS cancellation form

## Description of change

The form for cancelling an OMIS order has been reactified.

## Test instructions

Go to an OMIS order with a status of `draft` or `quote_awaiting_acceptance` and click the 'Cancel' link in the header. You should be able to cancel the order.

## Screenshots

### Before

<img width="468" alt="Screenshot 2023-10-26 at 11 41 46" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/271d3fe1-27d8-424f-afdc-a82bb25eb232">

### After

<img width="463" alt="Screenshot 2023-10-26 at 11 41 18" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/b13a11c1-df75-4fcb-9825-6b18a2579601">

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
